### ISSUE TITLE
Ar,js Crash  only Safari  Iphone 

### ISSUE THREAD

**Issue by yuber1792** [jeromeetienne/AR.js#588]
the safari browser restarts by allowing camera access

**To Reproduce**
Take a test by opening the example on my iphone
https://codepen.io/nicolocarpignoli/pen/vMBgob

Result:

![IMG_1840](https://user-images.githubusercontent.com/2119030/65071674-e856e580-d954-11e9-9727-e945eb39675f.PNG)

the camera loads and starts to zoom

![IMG_1841](https://user-images.githubusercontent.com/2119030/65071692-f0168a00-d954-11e9-8ecf-dca05667603b.PNG)

and then it goes blank and this message comes out

![IMG_1842](https://user-images.githubusercontent.com/2119030/65071814-2bb15400-d955-11e9-9da5-ec64e5403f6a.PNG)

![IMG_1844](https://user-images.githubusercontent.com/2119030/65071821-2f44db00-d955-11e9-8234-8a4607c413b7.PNG)

iOS : 12.4 

desktop safari on android chrome and desktop works fine, just pass this on on safari for iphone

I am using these scripts
<script src="https://aframe.io/releases/0.9.2/aframe.min.js"></script>
<script src="https://raw.githack.com/jeromeetienne/AR.js/1.7.7/aframe/build/aframe-ar.js"></script>

**Comment by nicolocarpignoli** [comment]
which version of safari? it happened also with old safari version, or you first try Ar.js now?

**Comment by nicolocarpignoli** [comment]
@yuber1792 can you please debug Safari and see what the error is on Console? are you able to do that?

**Comment by nicolocarpignoli** [comment]
Ok, I have done some tests. Definitely not an error with AR.js. It's a restriction imposed by Apple.

Please, go on 

`Settings -> Safari -> Camera & Microphone Access`

and activate the option if disabled.

**Comment by yuber1792** [comment]
[Uploading video-1569022384.mp4.zip…]()

**Comment by yuber1792** [comment]
[videooo.zip](https://github.com/jeromeetienne/AR.js/files/3638104/videooo.zip)

**Comment by nicolocarpignoli** [comment]
@yuber1792 this is a completely new thing I never saw before. Is your entire phone crashing?

Is the behaviour consistent and you can reproduced exactly very time?

Do you experience the same with http://nicolocarpignoli.github.io/ar-playground/index.html ?

**Comment by yuber1792** [comment]
hello, before it just went blank and restarted the browser

 http://nicolocarpignoli.github.io/ar-playground/index.html

That url works perfect, what version are you using?

**Comment by nicolocarpignoli** [comment]
mmmh don't think it's AR.js version, from that version (1.7.5) and the one I used (latest, 1.7.8) no big changes happened.

https://github.com/nicolocarpignoli/nicolocarpignoli.github.io/blob/master/ar-playground/index.html

this is the code above that works for you. let me know. I start to think it's something related to Codepen maybe?

**Comment by yuber1792** [comment]
No, in my example it didn't work either, I'm going. to review your example to see if it is solved.

**Comment by yuber1792** [comment]
hello look that I deploy with an ionic web project, that's the problem, I put the code in an angular project and it works.

**Comment by msob83** [comment]
Turn off  "Handoff" permission option on your Mac if you have iphone and mac on . It worked for me

**Comment by rondog** [comment]
I noticed it was only crashing while browsing in private mode in iOS safari. It appears from your screen shot that you are as well, albeit, still should not be crashing
